### ISSUE TITLE
gitian: Debian 9 doesn't ship with ifconfig

### ISSUE THREAD

**Issue by Sjors** [bitcoin-core/docs#15]
The instructions here won't work if `ifconfig` is not present:

https://github.com/bitcoin-core/docs/blob/master/gitian-building/gitian-building-setup-gitian-debian.md#setting-up-debian-for-gitian-building

Solution is to tell user to `sudo apt-get install net-tools` or change the instructions to use the `ip` command.

That still doesn't work for me. I had to `sudo -s` and then `source /etc/rc.local`.

(also, there's some inconsistency between this user being called `debian` vs `gitianuser`, but that creates a clear error message. However `gitian-builder/var/lxc.config` assumes `debian`)

**Comment by Sjors** [comment]
> also, there's some inconsistency between this user being called debian vs gitianuser

See #20

**Comment by Sjors** [comment]
This is a Debian 9 issue. It works fine in Debian 8.

Probably still worth addressing at some point. Can we get a "Debian 9" label? cc @fivepiece @laanwj @MarcoFalke.

**Comment by fivepiece** [comment]
We could use 
```
brctl addbr br0
ip addr add 10.0.3.2/24 brd 10.0.3.255 dev br0
ip link set br0 up
```
to set up the bridge instead of `ifconfig`.  I haven't tested it on a NAT'd host VM though.

**Comment by MarcoFalke** [comment]
See #22 for something that seems to work for me.

**Comment by Sjors** [comment]
Fixed through #22.
